client/src/main/java/io/pravega/client/batch/SegmentRange.java

```diff
@@ -28,13 +28,13 @@
      * Returns the stream name the segment is associated with.
      * @return The stream name.
      */
-    String getStreamName();
+    String getStream();
```

Comment:
I'm sorry for requesting yet another change, but I think we should make this consistent with the `Stream` interface. In the `Stream` interface, we have

```
getStreamName
getScopedName
```

In the case we only need to return the stream name (without scope), then we should call it `getStreamName`. `getScope` remains `getScope`, that's consistent with the `Stream` interface.

One question is whether we should also have `getScopedName` in this interface again to be consistent with the `Stream` interface.

Reply:
Sure, I have updated the apis to `getStreamName` and `getScope`.
`getScopedName` would cause a confusion on whether it includes the segment number, hence not planning to add it.